helm/dex-app/templates/dex-secret.yaml

```diff
@@ -32,6 +32,11 @@ stringData:
       - https://{{ .Values.Installation.V1.GiantSwarm.OIDC.ClientAddress }}/callback
       name: dex-k8s-authenticator
       secret: {{ .Values.Installation.V1.Secret.OIDC.ClientSecret }}
+    - id: zQiFLUnrTFQwrybYzeY53hWWfhOKWRAU
```

Comment:
This is the same as the Auth0 client ID

Reply:
Let's not bring any specific configuration data directly into the template.
You can add it into default draughtsman